src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/EntityAIRanger.java

```diff
@@ -121,22 +121,17 @@ protected int getAttackRange()
     private int getRealAttackRange()
     {
         int attackDist = BASE_DISTANCE_FOR_RANGED_ATTACK;
-        // + 2 Blockrange per building level for a total of +10 from building level
+        // + 1 Blockrange per building level for a total of +5 from building level
         if (buildingGuards != null)
         {
-            attackDist += buildingGuards.getBuildingLevel() * 2;
+            attackDist += buildingGuards.getBuildingLevel();
         }
         // ~ +1 each three levels for a total of +15 from guard level
         if (worker.getCitizenData() != null)
         {
             attackDist += (worker.getCitizenData().getLevel() / 50.0f) * 15;
         }
 
-        if (target != null)
```

Comment:
Oh no, I liked this, I think there should be an advantage depending on the height he stands on.

Reply:
hm it messes with your 2d distance calculations though which other stuff is based on

Reply:
What do you mean? I mean it only increases the max attack distance depending on the height.